opendcs-integration-test/src/test/resources/org/opendcs/odcsapi/res/it/DEFAULT/config_input_data.json

```diff
@@ -40,6 +40,10 @@
         }
       }
     ],
-    "formatStatements" : [ ]
+    "formatStatements" : [ {
+      "sequenceNum": 0,
```

Comment:
should we consider just using the decodesscript test format as the "intermediary representaion"? we do have some plans to allow other script engines for decoding in the future.

Reply:
I think that's a great idea. Probably worth doing after merging the repositories so we can take advantage of the same files.